DEVELOPMENT.md

```diff
@@ -76,7 +76,10 @@ Please refer to [Contributing via Pull Requests](./CONTRIBUTING.md#contributing-
 4. Start using `rush cinstall` or `rush cupdate` instead of `rush install` or `rush update`.
 
 ### Add packages using rush:
-1. `rush add -p <packageName> --caret -s; rush cupdate`
+1. Add package to 1 project: `cd <TO_PROJECT_DIR>; rush add -p <packageName> --caret -s; rush cupdate`
```

Comment:
maybe we add the `-m` flag since we'll get rush errors if packages have mismatched versions anyway?

Reply:
@bhudnell if we add the -m flag then it will update the other packages with the version you specified (if any) which might not be what you want. If you want to use the version that is being used in other packages then we have 2 options:
1. Specifying the exact version.
2. Don't specify the version and use the -m option. 
 
So I will leave this for users to decide

Reply:
sounds good!